### PR TITLE
ffsend: add Chinese translation

### PR DESCRIPTION
**Pull request by snf1k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/ffsend.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> 轻松和安全地分享文件。
+> 更多信息：<https://gitlab.com/timvisee/ffsend>.
+
+- 上传文件：
+
+`ffsend upload {{路径/到/文件}}`
+
+- 下载文件：
+
+`ffsend download {{url}}`
```
Comment (leonvsc) [inline pages.zh/common/ffsend.md:12]: Is `url` translatable to Chinese?

**Review by leonvsc — APPROVED** [review]
LGTM! I have one question left in the comments.

File: pages.zh/common/ffsend.md (hunk 1 above)
Comment (snf1k) [reply pages.zh/common/ffsend.md:12]: There is no suitable translation for the time being, if translated into Chinese it is 统一资源定位器(unified resource locator), but this is too long. 
Translated into 链接(link), but the link is not exactly equivalent to a URL

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.zh/common/ffsend.md (hunk 1 above)
Comment (kbdharun) [reply pages.zh/common/ffsend.md:12]: Not just Chinese, we don't translate acronyms throughout our pages, I guess I will add this to the tracker issue too.

**Pull request merged**